packages/strapi-admin/admin/src/plugins.js

```diff
@@ -23,15 +23,15 @@ window.strapi = Object.assign(window.strapi || {}, {
 });
 
 module.exports = {
-  'strapi-plugin-documentation': require('../../../strapi-plugin-documentation/admin/src').default,
+  // 'strapi-plugin-documentation': require('../../../strapi-plugin-documentation/admin/src').default,
   'strapi-plugin-users-permissions': require('../../../strapi-plugin-users-permissions/admin/src')
     .default,
-  'strapi-plugin-content-manager': require('../../../strapi-plugin-content-manager/admin/src')
-    .default,
-  'strapi-plugin-content-type-builder': require('../../../strapi-plugin-content-type-builder/admin/src')
-    .default,
+  // 'strapi-plugin-content-manager': require('../../../strapi-plugin-content-manager/admin/src')
+  //   .default,
+  // 'strapi-plugin-content-type-builder': require('../../../strapi-plugin-content-type-builder/admin/src')
+  //   .default,
 
-  'strapi-plugin-email': require('../../../strapi-plugin-email/admin/src').default,
-  'strapi-plugin-upload': require('../../../strapi-plugin-upload/admin/src').default,
-  'strapi-plugin-graphql': require('../../../strapi-plugin-graphql/admin/src').default,
+  // 'strapi-plugin-email': require('../../../strapi-plugin-email/admin/src').default,
+  // 'strapi-plugin-upload': require('../../../strapi-plugin-upload/admin/src').default,
+  // 'strapi-plugin-graphql': require('../../../strapi-plugin-graphql/admin/src').default,
```

Comment:
I think we should uncomment those lines 😆

Reply:
It does not really matter because this file is only used when developing in the monorepo.